facebook/Docusaurus
# feat: allow different host through CLI flag

## Motivation

This would allow folks running Docusaurus in a container to access the site on localhost.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan
From: https://github.com/facebook/Docusaurus/pull/920 which appears closed.

- Install a fresh Docuaurus in the host environment.
- Run `yarn start` in the host environment.
- Ensure the site is served at http://localhost:3000 as usual.
- Containerize the application (get it installed in a node based container)
- Run `yarn start` within the container.
- Confirm the site is not accessible at http://localhost:3000
- Stop the webserver (live reload).
- Apply the change from this merge request commit.
- Run `yarn start` within the container.
- Confirm the site is accessible at http://localhost:3000 

Note: I have tested this both locally and running inside a Docker container.

## Related PRs
https://github.com/facebook/Docusaurus/pull/920

Fixes #919 
(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)
